adoc/reference-webui-overview.adoc

```diff
@@ -99,6 +99,13 @@ The menu:Main Menu[Home > Notification Messages] page allows you to manage your
 
 image::home_notification_messages.png[scaledwidth=80%]
 
+You can configure the notifications lifetime in [path]``/etc/rhn/rhn.conf`` (the default lifetime is 30 days):
```

Comment:
Hrm. What does "lifetime" mean in this situation? Is it how long the notification remains visible for until it is manually cleared? Or something else? I'd like this to be more specific.

Reply:
So the wording should probably be something like this:

`You can configure the amount of time before notifications are deleted in [path]``/etc/rhn/rhn.conf`` (the default lifetime is 30 days, after which notifications are deleted from the database, regardless of read status):`

Reply:
I don't really like the parenthesis, but I'll leave that up to @keichwa to determine, based on context.